blockchain/blockchain.go

```diff
@@ -1682,14 +1682,13 @@ func (bc *BlockChain) insertChain(chain types.Blocks) (int, []interface{}, []*ty
 
 		if bc.cacheConfig.TrieNodeCacheConfig.NumFetcherPrefetchWorker > 0 {
 			// if fetcher works and only a block is given, use prefetchTxWorker
```

Comment:
Please change or remove the comment accordingly 😅

Reply:
Updated. Please take a look. :)